packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -274,11 +274,6 @@ class WebpackCLI {
                 alias: 'p',
                 pkg: '@webpack-cli/generators',
             },
-            {
-                name: 'migrate',
-                alias: 'm',
-                pkg: '@webpack-cli/migrate',
-            },
```

Comment:
Don't remove this, otherwise it will be breaking change

Reply:
done